test/unit/sql/remove-constraint.test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+/* jshint -W030, -W110 */
+const Support   = require(__dirname + '/../support');
+const current   = Support.sequelize;
+const expectsql = Support.expectsql;
+const sql = current.dialect.QueryGenerator;
+const dialect = Support.getTestDialect();
+
+if (dialect === 'mssql' || dialect === 'postgres') {
```

Comment:
No, mysql doesn't have `removeConstraint` in queryGenerator. We have to override `removeConstraint` in mysql queryInterface and it ultimately calls `QueryGenerator.removeIndexQuery` or `QueryGenerator.dropForeignKeyQuery` which already have tests covered